packages/store/src/patterns/exo-tools.js

```diff
@@ -267,7 +267,7 @@ harden(defendPrototype);
  * @param {Record<FacetName, KitContextProvider>} contextProviderKit
  * @param {Record<FacetName, Record<string | symbol, CallableFunction>>} behaviorMethodsKit
  * @param {boolean} [thisfulMethods]
- * @param {Record<string, InterfaceGuard>} [interfaceGuardKit]
+ * @param {Record<string, InterfaceGuard<any>>} [interfaceGuardKit]
```

Comment:
Please do the default-param-type thing. A type argument that adds no information should not need to be stated.